lib/iris/fileformats/netcdf/loader.py

```diff
@@ -236,7 +237,19 @@ def _get_cf_var_data(cf_var, filename):
             )
             # Get the chunking specified for the variable : this is either a shape, or
             # maybe the string "contiguous".
+            if CHUNK_CONTROL.mode == CHUNK_CONTROL.Modes.AS_DASK:
```

Comment:
```suggestion
            if CHUNK_CONTROL.mode == ChunkControl.Modes.AS_DASK:
```

See my comment about a class variable instead of an instance variable (and sorry for getting this wrong when I was demonstrating!).